src/api/user/user.router.ts

```diff
@@ -0,0 +1,90 @@
+import express, { NextFunction } from "express";
+import type { Request, Response } from "express";
+import { catchErrors, validateUuidParam } from "../../utils/middleware";
+import {
+  deleteUser,
+  getUser,
+  getUserBySystemId,
+  getUsers,
+  updateUser,
+  upsertUser,
+} from "./user.service";
+import { apiError } from "../../utils/types";
+
+export const userRouter = express.Router();
+
+/**
+ ** User Router Home
+ */
+userRouter.get(
+  "/",
+  catchErrors(async (req: Request, res: Response) => {
+    const users = await getUsers();
+    return res.status(200).json(users);
+  })
+);
+
+/**
+ ** Create new user
+ */
+userRouter.post(
+  "/create",
+  catchErrors(async (req: Request, res: Response) => {
+    const userData = req.body;
+    const newUser = await upsertUser(userData);
```

Comment:
Add validation for userData. Make sure the data is structured correctly and properties are provied